src/towncrier/test/test_write.py

```diff
@@ -226,3 +227,46 @@ def test_append_at_top_with_hint(self):
             output = f.read()
 
         self.assertEqual(expected_output, output)
+
+    def test_multiple_file_no_start_string(self):
+        """
+        Multiple file output with no start string works.
```

Comment:
Maybe move this to test_build to have better coverage  / end to end testing.

```suggestion
        When no `start_string` is defined, the generated content is added at the start of the file.
```